pkg/utils/common.go

```diff
@@ -163,6 +163,14 @@ func SyncData(log sdkTypes.KairosLogger, runner v1.Runner, fs v1.FS, source stri
 		}
 		if t, err := fs.RawPath(target); err == nil {
 			target = t
+			// create target path if it doesnt exists
+			if _, err := os.Stat(target); err != nil {
```

Comment:
looks like older vfs was more strict when dir were not there. Now its more relaxed and it doesnt fail so we need to check ourselves.